src/test/java/me/karakelley/http/handlers/StaticFilesHandlerTest.java

```diff
@@ -51,7 +57,13 @@ void testServesTextFile() {
       PublicDirectory publicDirectory = PublicDirectory.create(directory.toString());
       ContentPresenter contentPresenter = new HtmlPresenter();
       Handler handler = new StaticFilesHandler(publicDirectory, contentPresenter);
-      Response response = handler.respond(new Request(HttpMethod.GET, "/test1.txt",  "HTTP/1.1", null, null, 0));
+      Response response = handler.respond(new Request.Builder()
+                                          .setMethod(HttpMethod.GET)
```

Comment:
I'm not sure this indentation is correct... Run a "reformat code" before you merge this, por favor.